Celeste.Mod.mm/Content/Dialog/English.txt

```diff
@@ -22,6 +22,11 @@
 	POSTCARD_LEVELGONE= 		{big}Thank you ((player))!{/big}{n}But our {#ff1144}((sid)){#} is on another mountain.
 	POSTCARD_LEVELLOADFAILED= 	{big}Sorry!{/big}{n}But {#ff1144}((sid)){#} has failed loading.{n}Please send your {#44adf7}log.txt{#}{n}to the map builder.
 	POSTCARD_LEVELNOSPAWN= 		{big}Oops!{/big}{n}Someone misplaced your {#44adf7}starting point!{#}{n}Please check if your map{n}contains a {#44adf7}Player{#} entity.
```

Comment:
While we're in here, should we change this to "your map's starting room" instead of "your map"?

Reply:
Also maybe `player` should be lowercase, since that's the actual name of the entity in the map data.

Reply:
Personally I think it's pretty intuitive since the error occurs on loading into the first room. Plus I tried to avoid having to re-translate any of the old messages lol